src/app/utils/discord.server.ts

```diff
@@ -11,11 +10,8 @@ export type ExecuteWebhookData = RESTPostAPIWebhookWithTokenJSONBody &
         wait: true;
     };
 
-export async function sendDiscordWebhook(data: ExecuteWebhookData) {
-    await api.webhooks.execute(config.discord.webhookId, config.discord.webhookToken, data);
-}
-
 export async function sendDiscordWebhookWithUrl(url: string, data: ExecuteWebhookData) {
+    if (!url.length) return;
```

Comment:
It will actually fail earlier, as we're checking they exist in src/app/utils/config.server.ts

Maybe it should check if the URL matches the webhook pattern? This way we don't send random requests if someone doesn't set it up correctly. Or if they want to disable it in devel.

Reply:
Maybe, but if then it should be sth. which the config checks and not the `sendDiscordWebhookWithUrl` function.
But this would be a different PR.